src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1939,3 +1939,300 @@ def test_sql_subscription_usages(self):
+                 .format(rg, server, pool_name),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', pool_name),
+                     JMESPathCheck('state', 'Ready'),
+                     JMESPathCheck('edition', 'Premium'),
+                     JMESPathCheck('zoneRedundant', True)]).get_output_in_json()
+
+        # Test creating pool with zone resilience set to false.  Expect regular pool created.
+        self.cmd('sql elastic-pool create -g {} --server {} --name {} --edition {} --zone-redundant {}'
+                 .format(rg, server, pool_name_2, "Premium", False))
+
+        self.cmd('sql elastic-pool show -g {} --server {} --name {}'
+                 .format(rg, server, pool_name_2),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', pool_name_2),
+                     JMESPathCheck('state', 'Ready'),
+                     JMESPathCheck('edition', 'Premium'),
+                     JMESPathCheck('zoneRedundant', False)]).get_output_in_json()
+
+        # Test creating pool with no zone resilience set.  Expect regular pool created.
+        self.cmd('sql elastic-pool create -g {} --server {} --name {} --edition {}'
+                 .format(rg, server, pool_name_3, "Premium"))
+
+        self.cmd('sql elastic-pool show -g {} --server {} --name {}'
+                 .format(rg, server, pool_name_3),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', pool_name_3),
+                     JMESPathCheck('state', 'Ready'),
+                     JMESPathCheck('edition', 'Premium'),
+                     JMESPathCheck('zoneRedundant', False)]).get_output_in_json()
+
+    @ResourceGroupPreparer()
+    @SqlServerPreparer()
+    def test_sql_update_zone_resilient_pool(self, resource_group, resource_group_location, server):
+        pool_name = "updateToZonedPool"
+        pool_name_2 = "updateToUnzonedPool"
+        pool_name_3 = "updateNoParamForUnzonedPool"
+        pool_name_4 = "updateNoParamForZonedPool"
+
+        rg = resource_group
+
+        # Test running update on regular pool with zone resilience set to true.  Expect zone resilience to update to true
+        self.cmd('sql elastic-pool create -g {} --server {} --name {} --edition {}'
+                 .format(rg, server, pool_name, "Premium"))
+
+        self.cmd('sql elastic-pool show -g {} --server {} --name {}'
+                 .format(rg, server, pool_name),
+                 checks=[
+                     JMESPathCheck('resourceGroup', rg),
+                     JMESPathCheck('name', pool_name),
+                     JMESPathCheck('state', 'Ready'),
+                     JMESPathCheck('edition', 'Premium'),
+                     JMESPathCheck('zoneRedundant', False)]).get_output_in_json()
```

Comment:
`JMESPathCheck('zoneRedundant', False)` can be in basic pool test

Reply:
I see you using `get_output_in_json()` a lot, this is only needed if you actually need to access any of the resource's properties. In most places you don't need it